assignment2/src/problem2_2.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script type="text/javascript">
+        function getSession() {
+            document.write('Name = ${sessionStorage.getItem("name")} </br>');
```

Comment:
Use tick operator instead of single quotes as using template string. In this case, it will not pull value, it will whole as a string.